sw/airborne/Makefile

```diff
@@ -38,16 +38,16 @@ VPATH = .
 ifneq ($(MAKECMDGOALS),clean)
   include $(AIRCRAFT_BUILD_DIR)/Makefile.ac
   $(TARGET).srcs += $($(TARGET).EXTRA_SRCS)
-  include ../../conf/Makefile.local
+  include $(PAPARAZZI_HOME)/conf/Makefile.local
```

Comment:
Hm.. not quite sure if these makefiles should really be in `PAPARAZZI_HOME` (instead of the conf in `PAPARAZZI_SRC`).
@gautierhattenberger what do you think?
Probably doesn't matter that much right now as the separation is not really much used and probably rather untested, but IMHO only the conf xml files should be in HOME (and var).